deps/openssl/README.md

```diff
@@ -0,0 +1,76 @@
+This has a new binding scheme in builing OpenSSL-1.1.0 library with
+Node.js. OpenSSL-1.1.0 uses a new build system with perl for various
+supported platforms. See `openssl/Configurations/README` and
+`openssl/Configurations/README.design` in the OpenSSL source for
+details.
+
+In order to build OpenSSL library without perl in the build of Node.js
+for various supported platforms, platform dependent files (e.g. asm
+and header files ) are pre-generated and stored into the
+`config/archs` directory.
+
+- `config/Makefile` and `config/generate_gypi.pl`
+
+  Makefile has supported platform list and generates and copies
+  platform dependent files (e.g. asm files) into arch directory with
+  generate.pl.  Platform dependent gypi files also created obtaining
+  build information from `configdata.pm` that is generated with
+  `Configure` in the OpenSSL build system.
+
+  For Windows, `Configure` generates makefile that is only available to
+  nmake command.  `config/Makefile_VC-WIN32` and
+  `config/Makefile_VC-WIN64A` are made created by hand for the use of
```

Comment:
`are made created` -> `are made` or `are created`

Reply:
I fixed to use `are made`.